Makefile

```diff
@@ -246,9 +249,17 @@ ci-docker-build: ## Build docker image with the operator (without generating ass
 ci-test: vet envtest ## Run tests in CI (without generating assets)
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... $(TESTARGS) -coverprofile cover.out
 
-.PHONY: integration-test
-integration-test:  setup-vault ## Run integration tests for Vault OSS
+.PHONY: copy-config
+copy-config: ## Copy kustomize config to CONFIG_BUILD_DIR
+	@rm -rf $(CONFIG_BUILD_DIR)
+	@cp -a $(CONFIG_SRC_DIR) $(CONFIG_BUILD_DIR)
```

Comment:
May need a `mkdir -p $(CONFIG_BUILD_DIR)` in here somewhere?

Reply:
Good catch, fixed in https://github.com/hashicorp/vault-secrets-operator/pull/164/commits/c2c295d1d279c9c86301d09857d60c602fe1ce40